doc/templates/GTFS-RT-Config.md

```diff
@@ -5,46 +5,31 @@
    - Generated directory is: /doc/user 
 -->
 
-
-# Updater configuration
```

Comment:
You are now missing the top level header (line starting with `#`), is this on purpose?

Reply:
Yes, because mkdocs generates them.